### PR TITLE
Feature/newsletter signup

### PR DESCRIPTION
**Pull request by willblake01**
Draft version of newsletter signup.

- I had to comment out a few things to get the mailer preview to render with the data dump I have.

### CODE REVIEW

**Comment by ColinW520** [comment]
Mailer preview is a bit of a jerk sometimes. Been a weird issue between what works on production, and what words in dev/staging/test. Needs resolved; but alas, tech debt.

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -21,4 +21,5 @@
     %main.main
       = render partial: 'flash_messages', flash: flash
       = yield
+    = render partial: "shared/sections/newsletter_signup"
```
Comment (ColinW520) [inline app/views/layouts/application.html.haml:24]: 👍

File: app/views/layouts/mailer.html.haml [hunk 2]
```diff
@@ -16,8 +16,8 @@
             %table{:bgcolor => "#182B3F"}
               %tr
                 %td
-                  %a{href: root_url}
-                    %img{:src => asset_url('logo-text-inside.png'), style: "max-width: 100px;"}/
+                  -# %a{href: root_url}
+                  %img{:src => asset_path('logo-text-inside.png'), style: "max-width: 100px;"}/
```
Comment (willblake01) [inline app/views/layouts/mailer.html.haml:20]: I had to change this to asset_path to get it to render in development. I thought the action_mailer.default_url_options in development.rb should have made asset_url work but I may have misunderstood.

File: app/views/shared/sections/_newsletter_signup.haml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+%section
+  .container
+    .row.mt-5.mb-5.text-center
+      .col
+        %div{:style => "background-color:red;"}
```
Comment (willblake01) [inline app/views/shared/sections/_newsletter_signup.haml:5]: I just have the background red to dev, would remove once that padding issue is resolved.

**Comment by ColinW520** [comment]
Help me understand, what _is_ the issue here? Is this supposed to be full width? Look like part of the site?

**Pull request closed without merging**